core/chain-configs/src/client_config.rs

```diff
@@ -162,6 +162,24 @@ impl SyncConfig {
     }
 }
 
+#[derive(serde::Serialize, serde::Deserialize, Clone, Copy, Debug)]
```

Comment:
Should this be in near_config.config close to where `StateSyncConfig` is?

Reply:
Actually, it might be fine here as we need to plumb it through to chain. That apparently only comes from chain config which comes from client config...